app/controllers/notifications_controller.rb

```diff
@@ -0,0 +1,91 @@
+class NotificationsController < ApplicationController
+  before_action :show_notification_count, only: :index
+
+  def index
+    if current_user.user_type == "taskee" then taskee_notifications end
```

Comment:
Line 5 and 6 can be refactored to `send("#{current_user.user_type}_notifications")`.